README.md

```diff
@@ -5,24 +5,22 @@
 
 This is a set of tools and APIs for issuers of zk-proof credentials, designed to be extensible. It allows an authenticated user to create schemas for issuing and managing credentials of identities. It also provides a web-based [frontend (UI)](ui/README.md) to manage issuer schemas, credentials and connections.
 
-## Installation
-
-There are two options for installing and running the server alongside the UI.
+This repository is for anyone to create their own [issuer node](https://0xpolygonid.github.io/tutorials/issuer-node/issuer-node-overview/) for Polygon ID.
 
-### Option 1 - Using Docker only
+---
 
-Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
+## Installation
 
-#### Requirements for Docker-only
+There are two options for installing and running the server alongside the UI:
 
-- [Docker Engine](https://docs.docker.com/engine/) 1.27+
-- Makefile toolchain
-- Unix-based operating system (e.g. Debian, Arch, Mac OS X)
+1. [Docker Quick Start Guide](#docker-quick-start-guide)
+2. [Standalone Mode Guide](#standalone-mode-guide)
 
-_NB: There is no compatibility with Windows environments at this time._
+### Docker Quick Start Guide
 
-#### Setup for Docker-only
+Running the app with Docker allows for minimal installation and a quick setup. This is recommended **for evaluation use-cases only**, such as local development builds.
 
+**Full overview:**
```

Comment:
This isn't really an "overview". I would just omit this, since it isn't a header.

Reply:
Change this to TL;DR Steps to give the idea of what is being done and updated the link to the sections that were missing.